### PR TITLE
Add types for safe apps access control property and `clientUrl` param to the query

### PR DESCRIPTION
**Pull request by mikheevm**
This is needed for https://github.com/gnosis/safe-react/issues/2672. The properties are live on staging config service, the PR to the client gateway is still open.

Ready to merge after https://github.com/gnosis/safe-client-gateway/pull/774

### CODE REVIEW

File: src/types/safe-apps.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
+export type SafeAppNoRestrictionsPolicy = {
+  type: 'NO_RESTRICTIONS'
+}
+
+export type SafeAppDomainAllowlistPolicy = {
+  type: 'DOMAIN_ALLOWLIST'
```
Comment (katspaugh) [inline src/types/safe-apps.ts:6]: Might be worth exporting the possible types as an enum, so that in the react app you can branch based on the type?

**Review by katspaugh — APPROVED** [review]
Nice! Just one suggestion.

File: src/types/safe-apps.ts (hunk 1 above)
Comment (iamacook) [reply src/types/safe-apps.ts:6]: This would be my only suggestion as well.

Comment (mikheevm) [reply src/types/safe-apps.ts]: brainfart sers, thanks for noticing

**Review by iamacook — APPROVED** [review]
🚀

**Review by katspaugh — CHANGES_REQUESTED** [review]
There's some dump file committed.
Please also update the version in package.json so that I can publish the package.

**Review by katspaugh — APPROVED** [review]
Thhhhhhanks!

**Review by iamacook — APPROVED** [review]

**Pull request merged**